### DISCUSSION TITLE
Publishing a static website with Pyodide

### DISCUSSION THREAD

**Discussion by lawchin** [flet-dev/flet#1293] [General]
Hello, can somebody please create a youtube tutorial regarding "Publishing a static website with Pyodide".

full detail here:
https://flet.dev/docs/guides/python/publishing-static-website/

I don't know if I add a module (pyrebase4) is it still a pure python package

**Comment by FeodorFitsner** [comment]
I believe `pyrebase4` is a pure Python module.

If you download it's wheel from [here](https://pypi.org/project/Pyrebase4/#files) - it's just zip file - you'll see it contains no binary files, only `*.py` ones:

![image](https://user-images.githubusercontent.com/5041459/232171213-679dfc11-1311-4fe0-9752-33bb6957b3b9.png)
